### ISSUE TITLE
Can't operate partitioned collections inside Meteor.defer

### ISSUE THREAD

**Issue by skemb321** [mizzao/meteor-partitioner#15]
Hi mizzao I noticed that we can't operate on partitioned collections inside meteor.defer. Were you able to work it out prior to the issue you've posted on Meteor repo https://github.com/meteor/meteor/issues/2221 ? 

Please help. Thanks

**Comment by mizzao** [comment]
Yeah, I usually just store the group from `Partitioner.group()` and then `Partitioner.bindGroup` explicitly in the defer function.

This is because `DDP._CurrentInvocation.get()` is undefined in the defer block, so we don't know what user context it is in.

**Comment by skemb321** [comment]
nice! Thanks! I will try this.

**Comment by skemb321** [comment]
it worked! :dancers:

**Comment by mizzao** [comment]
Let's leave this open as a signpost so that people know about it and as something that we may be able to fix or improve in the future. This would affect `Meteor.defer`, `Meteor.setInterval`, and `Meteor.setTimeout`.

By the way, if you are using `Meteor.defer`, you may want to just consider the possibility of doing `this.unblock()` in the method and going on with whatever is happening in the defer block, so you don't have to avoid these gynmastics.
